vendor/blockchain_contracts/tests/htlc_harness/mod.rs

```diff
@@ -1,17 +1,21 @@
 use crypto::{digest::Digest, sha2::Sha256};
 use ethereum_support::{Address as EthereumAddress, ToEthereumAddress};
 use hex::FromHexError;
+use hex_literal::hex;
 use secp256k1_support::KeyPair;
 use std::{str::FromStr, thread::sleep, time::Duration};
 
 mod erc20_harness;
 mod ether_harness;
+pub mod timestamp;
```

Comment:
:wrench: To be consistent with the other modules, this should be `mod timestamp` and `pub use self::timestamp::Timestamp` at line 17.